.bazelrc.common

```diff
@@ -70,3 +70,7 @@ build --incompatible_strict_action_env
 
 # Enable with --config=release
 build:release --stamp --workspace_status_command=$(pwd)/workspace_status.sh
+
+# Config to enable building and testing with bzlmod
+build:bzlmod --enable_bzlmod
+build:bzlmod --//:flag_bzlmod
```

Comment:
Unfortunately we can't declare a `config_setting` on `--enable_bzlmod`, so we have to detect it for `select`s this way.

Reply:
That seems like a bug (I assume you tried it)

Reply:
Yep, I tried.